src/layout/layout.css

```diff
@@ -0,0 +1,10 @@
+.layout {
```

Comment:
Use  [react-bootstrap grid](https://react-bootstrap.netlify.com/layout/grid/). it now supports flexblox.

Reply:
fixed